lib/job-iteration/csv_enumerator.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "csv"
```

Comment:
Are we sure we want this? I guess this allowed people to use the gem without loading CSV from the standard library if they didn't need it…

Reply:
I can revert this but I'll have to add it to the `test/unit/enumerator_builder_test.rb`